docs/build/voting-dapp-tutorial.md

```diff
@@ -0,0 +1,1252 @@
+---
+title: dApp Tutorial
+description: Build a complete dApp on Kadena
+---
+
+import PageRef from '@components/PageRef'
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Building a voting dApp
+
+In its most basic form, Kadena blockchain is a digital ledger. This ledger is never stored, but rather exists on the “chain” supported by thousands of nodes simultaneously. Thanks to encryption and decentralization, Kadena blockchain’s database of transactions is incorruptible, and each record is easily verifiable. The network cannot be taken down or influenced by a single party because it doesn’t exist in one place.
+
+Blockchain can solve the many problems discovered in these early attempts at online voting. A blockchain-based voting application does not concern itself with the security of its Internet connection, because any hacker with access to the terminal will not be able to affect other nodes. Voters can effectively submit their vote without revealing their identity or political preferences to the public. Officials can count votes with absolute certainty, knowing that each ID can be attributed to one vote, no fakes can be created, and that tampering is impossible.
+
+---
+
+## Overview
+
+In this tutorial we will be building a voting application on the Kadena blockchain. The application will consist of three distinct layers:
+
+1. Smart contract (back-end)
+2. Web app (front-end)
+3. Gas Station
```

Comment:
I did a quick pass of the entire tutorial and, at first, I thought this section would be a workflow of how the tutorial would be organized. 

But the tutorial's main sections were actually:
`Environment Setup -> Smart Contracts -> Testing -> Deploy Contract -> Frontend -> Conclusion`

I think one approach would be to include a summary of what the big sections of the tutorial will be, and/or a Table of Contents with links to the different sections and sub-sections.